### ISSUE TITLE
Search

### ISSUE THREAD

**Issue by cassidyjames** [system76/pop-docs#20]
We're gonna want search eventually. For GitHub Pages and Jekyll support, something like this could work:

https://github.com/jekylltools/jekyll-tipue-search

**Comment by kylerschin** [comment]
Adding something. Have a menu bar with the main articles, followed by article search.

=======================================

      ICON               ICON                ICON
     blah blah       blah blah          blah blah

=============================================
 SEARCH SEARCH SEARCH SEARCH SEARCH
------------------------------------------------------------------------
ARTICLES ARTICLES ARTICLES ARTICLES ARTICLES
ARTICLES ARTICLES ARTICLES ARTICLES ARTICLES

**Comment by justxux** [comment]
@ahoneybun, Is this still relevant? It's kinda old.

**Comment by brs17** [comment]
> @ahoneybun, Is this still relevant? It's kinda old.

Yes it is @justxux , this issue is pointing out that there is no search functionality on this page: https://pop.system76.com/docs/ (where the docs from this repo are consumed).
